Make the multi lane view have lanes be of fixed width

krlvi opened the pull request:
This would of course require that at some point we put a
nice placeholder on the right side.

FYI @PavelLaptev  - let me know what you think, i was interested in trying this out

Commit:
Make the multi lane view have lanes be of fixed width

This would of course require that at some point we put a
nice placeholder on the right side.

krlvi commented:
@PavelLaptev I was about to merge this - but let me know if you hate this

PavelLaptev commented:
@krlvi to me, this doesn’t make sense—if you have only one branch, what’s the point of keeping it fixed?

krlvi commented:
Here's how it looks: 
<img width="1856" alt="image" src="https://github.com/user-attachments/assets/e15194c1-1714-4498-8644-fc74fb878ed7" />

I quite prefer it because fixed width gives me a sense of space

PavelLaptev committed:
css fixes

- make "fixed" line width really fixed
- stretch lane in "single" mode full width if there is only one lane

PavelLaptev commented:
To me, this looks a bit "unfinished"
